### PR TITLE
WIP add tests mocks and 508 tests for GIBCT

### PR DESCRIPTION
**Pull request by bshyong**

### CODE REVIEW

**Comment by bshyong** [comment]
Pausing on this until GIBCT is ready for production. These tests will not pass until the new gibct routes are visible in production

**Comment by bshyong** [comment]
This PR adds endpoint mocks for e2e and accessibility tests, and adds some minor changes to pass 508 tests on the search and results page

**Comment by bshyong** [comment]
@U-DON I think these will pass if we remove GIBCT from the ignore list. I believe the revproxy in production is still routing GIBCT requests to the standalone rails app, so AFAIK this should have no effect

**Comment by bshyong** [comment]
@U-DON Tests are passing in production now!  Did you add additional pages to the 508 tests here or should I do that?

**Comment by U-DON** [comment]
Awesome. I haven't yet, feel free to go add them.

**Comment by U-DON** [comment]
I wonder why the sitemap tests require extra time now?

**Comment by bshyong** [comment]
@U-DON I think it's because our GIBCT app takes time to load, so selenium is failing due to a short timeout. the sitemap spec hits the GIBCT app route now that I took GIBCT out of the ignorelist so it was failing out

**Comment by bshyong** [comment]
Extra pages have been added. These tests are now passing on all pages of GIBCT (landing, search results, profile)

File: test/accessibility/gibct-508.spec.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    client.timeoutsAsyncScript(2000);
+
+    client
+      .waitForElementVisible('body', Timeouts.normal)
+      .waitForElementVisible('.gi-app', Timeouts.slow)
+      .axeCheck('.main');
+
+    client
+    .clearValue('.keyword-search input[type="text"]')
+    .setValue('.keyword-search input[type="text"]', 'washington dc');
```
Comment (U-DON) [inline test/accessibility/gibct-508.spec.js]: Indent these two lines?

File: test/accessibility/gibct-508.spec.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    client
+      .click('#search-button')
+      .axeCheck('.main');
+
+    client
+      .click('.search-result a')
+      .axeCheck('.main');
```
Comment (U-DON) [inline test/accessibility/gibct-508.spec.js]: Should these last two checks wait for distinguishing elements to be visible?

File: test/accessibility/gibct-508.spec.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+    client.pause();
```
Comment (U-DON) [inline test/accessibility/gibct-508.spec.js]: Is it okay to leave it paused indefinitely? Other tests set a limit (2000 ms seems like the longest).

**Comment by bshyong** [comment]
These tests should pass once I resolve this bug: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2263

**Pull request merged**